lma-ai-stack/source/lambda_functions/share_meeting/requirements.txt

```diff
@@ -0,0 +1,4 @@
+# temporarily uncomment debugpy for local debugging
+# XXX
+# debugpy
+cryptography
```

Comment:
why do we need cryptography?

Reply:
For JWT token verification when a meeting is shared to check the Owner is authorized to perform the change.